plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -21,15 +21,29 @@ import {
   ReviewDecision,
   PullRequestsColumn,
   Author,
+  PRCardFormating,
+  Repository,
 } from './types';
 import { COLUMNS } from './constants';
 
 const GITHUB_PULL_REQUESTS_ANNOTATION = 'github.com/project-slug';
+const GITHUB_USER_LOGIN_ANNOTATION = 'github.com/user-login';
 
 export const getProjectNameFromEntity = (entity: Entity): string => {
   return entity?.metadata.annotations?.[GITHUB_PULL_REQUESTS_ANNOTATION] ?? '';
 };
 
+export const getUserNameFromEntity = (entity: Entity): string => {
+  return entity?.metadata.annotations?.[GITHUB_USER_LOGIN_ANNOTATION] ?? '';
```

Comment:
i see that you're just following precedent above, but i'll just note that it usually leads to worse bugs to have a `?? ''` fallback, than to explicitly return `string | undefined` and forcing the caller to explicitly handle that case.

Reply:
makes sense, will update both `getProjectNameFromEntity` and `getUserNameFromEntity` accordingly